Task 3/system/Coding/src/app/States/SearchingStartLineState.h

```diff
@@ -29,6 +33,11 @@
 */
 extern void SearchingStartLineState_enterStartTimer1AndStartDriving(void);
 
+/**
+ * @brief Searches for the startline with the line sensors
```

Comment:
Use imperative for doxygen documentation. See previous comments.

```suggestion
 * @brief Search for the startline with the line sensors.
```